packages/slate-editor/src/extensions/web-bookmark/types.ts

```diff
@@ -1,6 +1,12 @@
 import type { OEmbedInfo } from '@prezly/sdk';
+import type { BookmarkNode } from '@prezly/slate-types';
+
+import type { EmbedNode } from '#extensions/embed';
 
 export interface WebBookmarkExtensionParameters {
     withNewTabOption?: boolean;
+    withConversionOptions?: boolean;
     fetchOembed: (url: OEmbedInfo['url']) => Promise<OEmbedInfo>;
 }
+
+export type Presentation = `${typeof EmbedNode.TYPE}` | `${typeof BookmarkNode.TYPE}`;
```

Comment:
If using the type values everywhere is not an option, let's disconnect them then:

```suggestion
export type Presentation = 'embed' | 'card';
```